docs/spelling_wordlist.txt

```diff
@@ -250,8 +250,8 @@ classpaths
 cleartext
 cli
 clientId
-Cloudant
 cloudant
+CloudantV
```

Comment:
When I tried to add `CloudantV1` to the wordlist the check still failed claiming `CloundantV` wasn't in the list. The actual class name is `CloudantV1`, but the spell checker seems to interpret the string as `CloudantV` with the number 1 appended.

Reply:
Add `` around. (`CloundantV1`)